packages/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -293,42 +273,22 @@ export class Wizard extends React.Component<WizardProps, WizardState> {
   };
 
   componentDidMount() {
-    const { appendTo } = this.props;
-    const target: HTMLElement = this.getElement(appendTo);
-    if (this.isModal) {
-      if (this.container) {
-        target.appendChild(this.container);
-      }
-      this.toggleSiblingsFromScreenReaders(true);
+    const target = (typeof document !== 'undefined' && document.body) || null;
```

Comment:
Creative way to write a ternary.

Reply:
lol I guess I too quickly borrowed the logic from a different part of the code :P